build_pytorch_libs.sh

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
```

Comment:
Can we keep those scripts in `third_party`? I don't think they should be in the root, since they will generally be used by other scripts and not people

Reply:
Well, they also build things in `torch/lib` and `aten`. How about putting it in `tools`?

Reply:
`tools` sounds good